app/src/main/java/fr/free/nrw/commons/nearby/NearbyListFragment.java

```diff
@@ -74,29 +78,13 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
 
         // If we are returning here from a screen orientation and the AsyncTask is still working,
         // re-create and display the progress dialog.
-        if (isTaskRunning) {
+        if (isTaskRunning()) {
             progressBar.setVisibility(View.VISIBLE);
         }
     }
 
-    @Override
-    public void onSaveInstanceState(Bundle outInstanceState) {
-        // See http://stackoverflow.com/questions/8942135/listview-added-dublicate-item-in-list-when-screen-orientation-changes
```

Comment:
For preventing the duplication, I believe doing adapter.clear() in initialization work instead.